src/Domain.LinnApps/Suppliers/Exceptions/SupplierException.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Linn.Purchasing.Domain.LinnApps.Suppliers.Exceptions
+{
+    using System;
+
+    public class SupplierException : AppDomainUnloadedException
```

Comment:
Was AppDomainUnloadedException the intended type?

Reply:
no! good spot